app/src/pages/ExploreDataPage.tsx

```diff
@@ -54,39 +51,75 @@ function ReportWrapper(props: { madLib: MadLib; setMadLib: Function }) {
     });
   }
 
-  switch (props.madLib.id) {
-    case "diabetes":
-      // TODO we should add type safety to these instead of casting.
-      variableId = getPhraseValue(props.madLib, 1) as VariableId;
-      return <TellMeAboutReport variable={variableId} />;
-    case "compare":
-      variableId = getPhraseValue(props.madLib, 1) as VariableId;
+  switch (props.madLib.id as MadLibId) {
+    case "disvargeo":
       return (
-        <CompareStatesForVariableReport
-          stateFips1={getPhraseValue(props.madLib, 3)}
-          stateFips2={getPhraseValue(props.madLib, 5)}
-          variable={variableId}
+        <VariableDisparityReport
+          dropdownVarId={getPhraseValue(props.madLib, 1) as DropdownVarId}
+          stateFips={getPhraseValue(props.madLib, 3)}
         />
       );
-    case "dump":
-      return <ChartDumpReport />;
-    case "covid":
-      variableId = getPhraseValue(props.madLib, 1) as VariableId;
+    case "disvarcompare":
+      const compareDisparityVariable = getPhraseValue(
+        props.madLib,
+        1
+      ) as DropdownVarId;
       return (
-        <CovidReport
-          variable={variableId}
+        <Grid container spacing={1} alignItems="flex-start">
```

Comment:
In the side-by-side report the vega snowman menu seems to be cut off: https://screenshot.googleplex.com/8hWCzLsgJUaNfmw

Reply:
I actually fixed that in a follow up PR with some other UI changes :)